portfolio/src/main/webapp/index.html

```diff
@@ -55,13 +55,12 @@ <h3>Overview</h3>
                 art. I am no longer trying to pursue a career in game development, but I am exploring other areas in computer science, such as systems. I hope to learn a lot about web dev
                 from STEP!
                 </p>
-                <br>
-                <div class="container">
-                  <p>Here are fun facts about me:</p>
+                <p>Here are fun facts about me:</p>
+                <div class="container-fluid">
                   <div class="container" id="fact-container">
-                    <p>I am the oldest out of three kids</p>
```

Comment:
This might still be nice to have in the HTML.
 - Either for robots crawling for content to surface (the snippets on Google)
 - For users who don't have JS enabled

Reply:
Would that mean putting it in <script>?